cli/lib/nf3.mjs

```diff
@@ -666,6 +699,38 @@ class Nf3 {
     return res.data.commitments;
   }
 
+  /**
+  Returns the pending withdraws commitments
+  @method
+  @async
+  @returns {Promise} This promise resolves into an object whose properties are the
+  addresses of the ERC contracts of the tokens held by this account in Layer 2. The
+  value of each propery is an array of commitments originating from that contract.
+  */
+  async getPendingWithdraws() {
+    const res = await axios.get(`${this.clientBaseUrl}/commitment/withdraws`);
```

Comment:
Is it possible to move the functionality of getPendingWithdraws to the endpoint rather than to the SDK? The SDK should only filter per pkd. What do you think?

Reply:
Yes, I've refactored this doing all the work in the endpoint.